go.mod

```diff
@@ -9,11 +9,11 @@ require (
 	github.com/gertd/go-pluralize v0.1.1
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v0.1.0
-	github.com/google/go-cmp v0.3.0
+	github.com/google/go-cmp v0.4.0
 	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/client_golang v1.7.1
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.5.1
```

Comment:
please undo the changes to `go.mod` and `go.sum`. They are not germane to this pull request's content. If we want to update these packages, let's do that in a separate commit that is specific to the new code brought in with those new versions.